tests/unit/button/button_options.js

```diff
@@ -8,14 +8,14 @@ module( "button: options" );
 test( "disabled, explicit value", function() {
 	expect( 9 );
 
-	var element = $( "#radio01" ).button({ disabled: false });
+	var element = $( "#button" ).button({ disabled: false });
 	deepEqual( element.button( "option", "disabled" ), false, "disabled option set to false" );
 	deepEqual( element.prop( "disabled" ), false, "element is disabled" );
 
 	ok( !element.button( "widget" ).hasClass( "ui-state-disabled" ), "element gets ui-state-disabled" );
```

Comment:
The message here and on the next assertion doesn't match the assertion. I'd just drop the message arguments.